docs/snippets/snippets.toml

```diff
@@ -172,6 +172,11 @@ quick_start = [ # These examples don't have exactly the same implementation.
   "py",
   "rust",
 ]
+"archetypes/image_send_columns" = [ # This mixes `log` and `send_columns`. Since `log` is suspect to delays by the batcher, this test gets flaky.
```

Comment:
I wonder if this could be fixed by tuning the batcher parameters… but probably it's not possible since it is running on a separate thread.